src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,11 @@
+const errorHandler = ((err , req , res , next) => {
+    console.log('Error', err);
+    res.send(
+        {
+            error : err.error,
+            code : err.code,
+            message : err.error || err.message,
+            timestamp : new Date(),
+        });
```

Comment:
keep all these in a single variable and pass that in the response

Reply:
done

Reply:
> Have you really run this code, was this running properly? As I am unable to see any changes in your package.json file.

yes mam it was running properly